frontend/src/pages/destructionlist/review/DestructionListReview.tsx

```diff
@@ -217,19 +257,46 @@ export function DestructionListReviewPage() {
             fields={listModalFormFields}
             onSubmit={onSubmitDestructionListForm}
             validateOnChange
-            labelSubmit={zaakSelectionCount > 0 ? "Beoordelen" : "Accoderen"}
+            labelSubmit={zaakSelection.length > 0 ? "Beoordelen" : "Accoderen"}
           />
         </Body>
       </Modal>
+
       <DestructionListComponent
         storageKey={destructionListReviewKey}
-        zaken={zaken}
+        zaken={_zaken}
         selectedZaken={selectedZaken}
-        labelAction={zaakSelectionCount > 0 ? "Beoordelen" : "Accoderen"}
+        labelAction={zaakSelection.length > 0 ? "Beoordelen" : "Accoderen"}
         title={`${list.name} beoordelen`}
         onSubmitSelection={() => setListModalDataState({ open: true })}
         onSelect={onSelect}
         allowSelectAll={false}
+        actions={[
+          {
+            children: <Outline.ChatBubbleBottomCenterIcon />,
+            tooltip: tooltipMotivation && (
+              <>
+                <H2>Opmerking</H2>
+                <P>{tooltipMotivation}</P>
+              </>
+            ),
+            onMouseEnter: (_, detail) => {
```

Comment:
Not sure about this state machine setup but I can't think of a better appraoch right now. Ideally the state machine (if needed) should be next to the tooltip implementation. But since the motivation is quite specific making this generic would be a challenge...

I guess this is OK for now...